src/decoder/serde.rs

```diff
@@ -713,6 +719,18 @@ impl<'de> Deserialize<'de> for JavaScriptCodeWithScope {
     }
 }
 
+impl<'de> Deserialize<'de> for Binary {
```

Comment:
This follows the pattern of the other deserializers defined here. Since the deserialize implementation for Bson implements deserializing from extjson, we piggyback off that to either get a Binary or return an error.